test-suite/quantlibbenchmark.cpp

```diff
@@ -195,7 +196,7 @@ namespace {
             BenchmarkResult()  {
                 boost::unit_test::framework::register_observer(*this);
             }
-            ~BenchmarkResult() override {
+            virtual ~BenchmarkResult() override {
```

Comment:
`override` already implies `virtual` but it's ok, the clang-tidy CI run will probably change it automatically.

Reply:
I'll fix this and will rebase on top of current master